lib/spack/spack/cmd/create.py

```diff
@@ -435,6 +452,10 @@ def __call__(self, stage, url):
         """Try to guess the type of build system used by a project based on
         the contents of its archive or the URL it was downloaded from."""
 
+        if url is None:
+            self.build_system = 'bundle'
+            return
```

Comment:
It would be good to know more about how people "typically" create a new package from scratch.  It wouldn't be a big change to only create a `BundlePackage` only when explicitly requested.

Reply:
I think most people will provide the URL but I suspect when they do not provide the URL, they most often do not mean BundlePackage.  e.g., I think that `spack create -n foobar` should continue to create a `Package` subclass like it does now, and `spack create -n foobar -t bundle` should create a `BundlePackage`.

Reply:
Okay.

Reply:
Okay.